### DISCUSSION TITLE
groupSlug/groupName variable substitution not working for branch names

### DISCUSSION THREAD

**Discussion by fgblomqvist** [renovatebot/renovate#10813] [Request Help]
We have configured some paths like this:
```
"groupName": "foo-{{parentDir}}",
"groupSlug": "foo-{{parentDir}}",
```
While the grouping works, the branch comes out named `renovate/<stuff>foo-parentdir`. As in, the variable `parentDir` isn't substituted. I think this is a bug?

**Comment by rarkins** [comment]
Looks likely that those fields aren't template compiled so then the special chars get stripped out and the string converted to lower case.

**Reply by rarkins** [reply]
I think we define somewhere which fields are template compatible, so next step is to check if they're in there

**Reply by fgblomqvist** [reply]
I believe they are: https://docs.renovatebot.com/templates/

**Reply by rarkins** [reply]
Actually I remember wrong. That's listing the fields you can use in a template but not which fields can be templated. Anyway should be ok to add a missing compile somewhere but we probably need a reproduction repo to debug it.
